gh-pages/_docs/04-04-scmlogparser.md

```diff
@@ -30,15 +32,15 @@ The names of authors are saved when the --add-author flag is set.
 
 | SCM | Log format          | Command for log creation               | tracks renames | ignores deleted files | supports code churn |
 | --- | ------------------- | -------------------------------------- | -------------- | --------------------- | ------------------- |
+| SVN | SVN_LOG             | `svn log --verbose`                    | yes            | yes                   | no                  |
 | git | GIT_LOG             | `git log --name-status --topo-order`   | yes            | yes                   | no                  |
 | git | GIT_LOG_NUMSTAT     | `git log --numstat --topo-order`       | yes            | no                    | yes                 |
 | git | GIT_LOG_NUMSTAT_RAW | `git log --numstat --raw --topo-order` | yes            | yes                   | yes                 |
 | git | GIT_LOG_RAW         | `git log --raw --topo-order`           | yes            | yes                   | no                  |
```

Comment:
not included in SVNLogParser. Also I think this document can be renamed as well from scmlogparser to svnlogparser